docs/querying/raw-queries.md

```diff
@@ -0,0 +1,702 @@
+
+:::
+
+### `sql.col`
+
+:::caution
+
+This function is available for backwards compatibility, and there are currently no plans to deprecate it, 
+but it is not recommended to use in new code. Prefer instead to use [`sql.attribute`](#sqlattribute), [`sql.identifier`](#sqlidentifier),
+and the `sql` template tag.
+
+:::
+
+This function is a third way to reference a column name. It's similar to [`sql.identifier`](#sqlidentifier), but gives special meaning to the `*` characters.
+
+Here are a few examples:
+
+| Input     | `sql.col`   | `sql.identifier` |
+|-----------|-------------|------------------|
+| `*`       | `*`         | `"*"`            |
+| `users.*` | `"users".*` | `"users.*"`      |
+
+Unlike [`sql.attribute`](#sqlattribute), this method does not support any other special syntax, and does not map its input to a column name.
+
+### `sql.jsonPath`
+
+This function can be used to extract a JSON property from a JSON value
+
+```ts
+sequelize.query(sql`
+  SELECT ${sql.jsonPath(sql.identifier('data'), ['addresses', 0, 'country'])} AS country
+  FROM users
+`);
+```
+
+```sql
+-- postgres
+SELECT data#>ARRAY['addresses', '0', 'country'] AS country FROM users
+-- other dialects
+SELECT JSON_EXTRACT(data, '$.addresses[0].country') AS country FROM users
+```
+
+This can be useful to generate a JSON extraction query dynamically in a safe way.
+
+The JSON Path array accepts a mix of strings and numbers. If a string is used, it will be treated as a property name (used to access an object property).
+If a number is used, it will be treated as an index (used to access an array element).
+
+Make sure to use the correct type for your use case, as using the string `'0'` will try to access the property named `'0'` instead of the first element of the array.
+
+Read more about this feature in the [JSON Extraction](./select-in-depth.md#json-extraction) chapter.
+
+:::info
+
+Attributes support a shorthand syntax for JSON extraction. See [Casting syntax in `sql.attribute`](#use-the-json-extraction-syntax) for more information.
+
+:::
+
+### `sql.unquote`
+
+The `sql.unquote` function is used to execute the `JSON_UNQUOTE` (or equivalent) function on a JSON value:
+
+```ts
+sequelize.query(sql`
+  SELECT ${sql.unquote(sql.jsonPath(sql.identifier('data'), ['addresses', 0, 'country']))} AS country
+  FROM users
+`);
+```
+
+```sql
+-- postgres (the #>> operator unquotes, unlike the #> operator)
+SELECT data#>>ARRAY['addresses', '0', 'country'] AS country FROM users
+-- other dialects
+SELECT JSON_UNQUOTE(JSON_EXTRACT(data, '$.addresses[0].country')) AS country FROM users
+```
+
+Read more about this feature in the [JSON Extraction](./select-in-depth.md#json-extraction) chapter.
+
+### `sql.fn`
+
+This function exists for backwards compatibility with older versions of Sequelize but is not recommended for new code, as `sql` can be used to write
```

Comment:
I would say no. Using `col` is discouraged because it usually is a bad idea (it does not map the attribute name to the column name). `fn` is needlessly verbose, but it does not hurt anyone.